README.md

```diff
@@ -30,6 +30,8 @@ Upgrading from v4.x? Facebook PHP SDK v5.x introduced breaking changes. Please [
 Simple GET example of a user's profile.
 
 ```php
+require_once __DIR__.'/vendor/autoload.php'; // change path as needed
```

Comment:
Thanks @dojobo! Can you add spaces to keep it consistent with the other concat syntax? :)

```php
require_once __DIR__ . '/vendor/autoload.php';
```